buildchain/buildchain/targets/remote_image.py

```diff
@@ -101,6 +112,15 @@ def task(self) -> types.TaskDict:
             })
         return task
 
+    def _docker_pull(self) -> List[str]:
```

Comment:
No, we don't want to use `docker` directly anymore.

We should use the API, @jbertran did the work.

You can restore the code that was removed from my PR.

Reply:
See 396d60824825b9fb7792d1e18e8be3504fc2c24e